cdp_scrapers/instances/portland.py

```diff
@@ -144,6 +211,19 @@ def get_supporting_files(
         """
         Return SupportingFiles for a given EventMinutesItems
 
+        # Find title
```

Comment:
Ohhhh I am so so so sorry Shak. I thought this blob was code. You added to the docstring. I am not sure how I feel about it, because this is only a part of what the method does to get files. I can't decide.

Reply:
I gave this some more thought. And I personally think we should remove this blob you added to the docstring Shak. Because this just isn't the complete picture of what this method does, to scrape the supporting files. I like that we want to explain in easy-to-understand fashion what the method does. i.e. It scrapes this, then that, and so on. I think we will do so in a [Notes](https://numpydoc.readthedocs.io/en/latest/format.html#notes) section.

What we have here doesn't really help because it is an incomplete explanation of this method. What do you think? I think we should remove lines 216 - 228. We will in the near future add a Notes section to this method's docstring.

Reply:
Please don't forget to remove [lines 216 - 228](https://github.com/Shak2000/cdp-scrapers/blob/3643a2f6a389fdb6caf26d11b3b9032839e6baf1/cdp_scrapers/instances/portland.py#L216-L228) in this docstring (if you are OK with [my comment](https://github.com/CouncilDataProject/cdp-scrapers/pull/70/files#r791337575) of course). :sweat_smile: I think this is just confusing and doesn't actually help the reader, because it is not a correct description of what this method is doing. I will add to this docstring after this PR is merged.

Reply:
Done.

Reply:
I removed lines 216–228.